src/media/css/feed/feed.styl

```diff
@@ -182,6 +183,13 @@
             text-overflow: ellipsis;
             white-space: nowrap;
         }
+        .desc {
+            font-size: 16px;
+            font-weight: 200;
+            line-height: 18px;
+            margin: 5px auto 0;
+            max-width: $mobile-tile-width - $horizontal-tile-padding;
```

Comment:
Its width is $desktop-content due to positioning. Maybe that needs to be looked at but it seemed like a shitty time for potential regressions.